src/components/work/reservationbutton/ReservationButton.js

```diff
@@ -167,6 +167,26 @@ export const ReservationButton = ({
     " " +
     (isOnlineTranslated || workTypeTranslated);
 
+  async function handleOpenLoginAndOrderModal() {
+    //add order modal to store, to be able to access when coming back from adgangsplatform/mitid?
+    const uid = modal.save("order", {
+      pids: pids,
+      selectedAccesses: allEnrichedAccesses,
+      workId: workId,
+      singleManifestation: singleManifestation,
+    });
+    //open actual loginmodal
+    openLoginModal({
+      modal,
+      mode: LOGIN_MODE.ORDER_PHYSICAL,
```

Comment:
ligger alle deher parametre ikke i den modal du saver lige ovenfor ?

Reply:
yes. 
lige nu er der to forskellige måder at åbne bestil-modalen efter login for _FFU uden adgangsplatform_ og _bib med adgangsplatform_ 
FFU'er har brug for all den info for at kunne åbne bestil-modalen pt. Det burde jeg måske streamline...